modules/mcagar/src/test/ts/browser/api/TinyLoaderTest.ts

```diff
@@ -37,3 +37,19 @@ UnitTest.asynctest('TinyLoaderTest', (success, failure) => {
     base_url: '/project/tinymce/js/tinymce'
   }, success, failure);
 });
+
+UnitTest.asynctest('TinyLoader.setupInBodyAndShadowRoot passes logs through', (success, failure) => {
```

Comment:
Might need to add a `ShadowDom.isSupported()` check here, as this fails on PhantomJS/IE since it doesn't support shadow dom, so it only ever does the `body` run.

Reply:
Fixed.